src/Select/index.stories.js

```diff
@@ -136,6 +136,43 @@ CustomTypeahead.parameters = {
   },
 };
 
+export const ChangingTypeaheadBehavior = Template.bind({});
+ChangingTypeaheadBehavior.args = {
+  label: "Select an Industry",
+  getTypeaheadString: (userInput, selectItem) => {
+    console.info({ userInput });
+    console.info({ selectItem });
+
+    let searchString =
+      selectItem.props.searchValue || selectItem.props.value || "";
+
+    // user is searching by code
+    if (/\d/.test(userInput)) {
+      searchString = selectItem.props.value;
+    }
+
+    return searchString;
+  },
```

Comment:
It's not a true fuzzy search, but exposing this function to consumers to figure out what to do is the easiest, lowest risk change to allow typeahead by both NAICS code and sector name